W-19706583 urls to allow (#503)
mulesoft/docs-code-builder: modules/ROOT/pages/urls-to-allow.adoc

= URLs to Allow to Run Anypoint Code Builder

> Allowing URLs used by Anypoint Code Builder in your client OS security settings is important so that Anypoint Code Builder can interact with required external and Anypoint Platform hosts.

Reviewer: ```suggestion
Allow Anypoint Code Builder URLs in your client OS security settings to let the IDE work with the external and Anypoint Platform hosts.
```

== Allow Anypoint Code Builder URLs

> Follow these steps to allow URLs to ensure the best performance in Anypoint Code Builder.

Reviewer: ```suggestion
Allow URLs to ensure the best performance in Anypoint Code Builder.
```
